ETBezierHandle.m

```diff
@@ -16,10 +16,12 @@ @implementation ETBezierHandle
 
 - (id) initWithActionHandler: (ETActionHandler *)anHandler
            manipulatedObject: (id)aTarget
+		  objectGraphContext: (COObjectGraphContext *)aContext
                     partcode: (ETBezierPathPartcode)partcode
 {
 	self = [super initWithActionHandler: anHandler
-	                  manipulatedObject: aTarget];
+	                  manipulatedObject: aTarget
+					 objectGraphContext: aContext];
```

Comment:
Would be better to use space to align method keywords, otherwise they won't appear correctly aligned with a custom tab size like in Github.

I know it's annoying since Xcode always inserts tabs instead :(

In the long run, I'll probably convert EtoilePaint to use spaces instead of tabs, for a better Xcode compatibility.